neptune/alpha/internal/init_impl.py

```diff
@@ -203,10 +203,14 @@ def init(
     if capture_stderr and not _experiment.exists(stderr_path):
         _experiment.define(stderr_path, StringSeries([]))
 
-    if source_files is None:
-        if not is_ipython() and os.path.isfile(sys.argv[0]):
-            _experiment["source_code/files"].save_files(sys.argv[0])
-    else:
+    entrypoint = sys.argv[0] or None if not is_ipython() else None
```

Comment:
If source_files is not None and entrypoint file is in source_files then it should be path relative to source_files root (not to cwd).